Assets/GothicVR/Scripts/Player/Camera/CameraFade.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GVR
+{
+    public class CameraFade : MonoBehaviour
+    {
+        public Image cameraFadeImage;
+
+        private void Start()
+        {
+            Fade(0.1f, 0);
```

Comment:
Magic number.
Can you make this 0.1f a const variable on top of the class, please. Easier to update if we need it in the future.